code/Makefile

```diff
@@ -70,20 +85,23 @@ override stackArgs += --ghc-options="$(GHCFLAGS)"
 NOISY=no
 SUMMARIZE_TEX=no
 
-all: test
+# actual targets
+all: test graphs
```

Comment:
Just ensuring this change is intentional: this update will generate graphs every time `make` is invoked with no arguments.

Reply:
Yes, I want them done all the time.